src/Builder/SplitItemBuilder.php

```diff
@@ -171,12 +158,12 @@ public function build() : SplitItem
 
     public function getCheckboxStyle() : CheckboxStyle
     {
-        return $this->checkboxStyle;
+        return $this->menu->getCheckboxStyle();
     }
 
     public function setCheckboxStyle(CheckboxStyle $style) : self
```

Comment:
Can we drop the setters?

Reply:
Sorry and the `modify*` methods.

Reply:
`SplitItemBuilder::setCheckboxStyle()` is called from `CliMenuBuilder::addSplitItem()` so the split item can inherit the parent styles.